src/app/components/map-item-bar/map-item-bar.component.ts

```diff
@@ -78,21 +78,36 @@ export class MapItemBarComponent implements OnInit, OnDestroy {
     return item.FormNames.join(', ');
   }
 
+  getAttachmentsCount(count: number): number {
+    return count > 1 ? count - 1 : null;
+  }
+
+  handleMissingImage() {
+    this.firstAttachment = './assets/images/broken-image-w-bg.svg';
+  }
+
+  private isAttachments(url: string) {
+    if (url) {
+      return this.sanitizer.bypassSecurityTrustUrl(url);
+    } else {
+      this.showImage = false;
```

Comment:
Denne funksjonen gjør ting den ikke sier noe om at den skal gjøre. Se min forrige kommentar om this.showImage